Automattic/jetpack
# Protect card: Add the Last scan time (with tooltip) based on plugin state (uninstalled, Active)

This PR is the first PR in a series that will add/show various pieces of information on the Protect product card in My Jetpack. In this PR we will show:
1. if the Protect plugin is Not active, we will show the number of installed plugins and themes on the site.
2. If the Protect plugin is active, we will show the last scan time, i.e.- "17 hours ago".






## Proposed changes:

- Adds a 'themes' property to the `window.myJetpackInitialState` object.
- Imports/uses the `protect-status` package.

### Other information:


## Jetpack product discussion


Project Thread: pbNhbs-aP6-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Run this PR locally (Docker dev environment) or from a Jurassic Ninja site using the Beta Tester plugin running this branch.
- First make sure the Protect plugin is not active.
- Go to My Jetpack and view the Protect product card.
- Verify you see the number of installed plugins and themes of the site on the Protect cart (upper right corner).
- Hover over the info icon and verify the info popup works correctly and it matches the Figma design: https://www.figma.com/design/CSXFCR5NSlKh3i8Uny2ntV/Add-value-to-the-active-Protect-product-card-on-My-Jetpack?node-id=6584-12732&t=GZUnLc96j6xEtp3G-0
- Activate the Protect plugin.
- Go back to My Jetpack and view the Protect card again (while the Protect plugin is active).
- Verify you now see the "Last scan time:" with a proper value (in the upper right corner)
- You can verify and compare the design with the Figma mockup: https://www.figma.com/design/CSXFCR5NSlKh3i8Uny2ntV/Add-value-to-the-active-Protect-product-card-on-My-Jetpack?node-id=6581-2252&t=GZUnLc96j6xEtp3G-0
